aio/aio-proxy/aio_proxy/search/index.py

```diff
@@ -1,59 +1,6 @@
 import os
 
-from elasticsearch_dsl import (
-    Date,
-    Document,
-    Float,
-    Integer,
-    Keyword,
-    Text,
-    analyzer,
-    token_filter,
-    tokenizer,
-)
-
-# Define filters
-french_elision = token_filter(
-    "french_elision",
-    type="elision",
-    articles_case=True,
-    articles=[
-        "l",
-        "m",
-        "t",
-        "qu",
-        "n",
-        "s",
-        "j",
-        "d",
-        "c",
-        "jusqu",
-        "quoiqu",
-        "lorsqu",
-        "puisqu",
-    ],
-)
-french_stop = token_filter("french_stop", type="stop", stopwords="_french_")
-french_stemmer = token_filter("french_stemmer", type="stemmer", language="light_french")
-# ignore_case option deprecated, use lowercase filter before synonym filter
-french_synonym = token_filter(
-    "french_synonym", type="synonym", expand=True, synonyms=[]
-)
-
-# Define analyzer
-annuaire_analyzer = analyzer(
-    "annuaire_analyzer",
-    tokenizer=tokenizer("icu_tokenizer"),
-    filter=[
-        "lowercase",
-        french_elision,
-        french_stop,
-        "icu_folding",
-        french_synonym,
-        "asciifolding",
-        french_stemmer,
-    ],
-)
+from elasticsearch_dsl import Document
```

Comment:
How come you delete all this logic ? Is it not relevant anymore ?

Reply:
Apparently, adding the detailed mapping or modifying it does not matter here, because the mapping cannot me changed. The class is only used to indicate which index to use for searching.
I tested with, without and with a modified mapping and the results are the same.